jfix-stdlib-id-generator/src/main/kotlin/ru/fix/stdlib/id/generator/BitsConfiguration.kt

```diff
@@ -0,0 +1,35 @@
+package ru.fix.stdlib.id.generator
+
+class BitsConfiguration(
+        val serverPartBits: Int,
+        val timePartBits: Int,
+        val counterPartBits: Int
+) {
+
+    val serverPartMaxNumber: Long
+    val timePartMaxNumber: Long
```

Comment:
it is common practice to name such variables timePartMask  
mask means that this is set of 1111ones
maxNumber suggests that it could be 1010. In our case we can are working with masks that also in same time act as max numbers. But not vise versa.